lib/sirius_api/strategies/local_token.rb

```diff
@@ -15,11 +15,14 @@ def store?
 
       def authenticate!
         if access_token.blank?
-          fail 'Missing access token'
-          return
+          return 'Missing access token'
```

Comment:
I see. I believe it's a useful convention so I'd prefer if we could stick with it. With Warden you [throw a symbol](https://github.com/hassox/warden/wiki/Failures), failure should be handled in a separate middleware.
So to reiterate, strategies should `throw :warden` to maintain cascade, but bang methods in API helper should `raise` an authentication /authorization error.